src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -32,4 +32,10 @@ void CaptureViewElement::OnDrag(int x, int y) {
   RequestUpdate();
 }
 
+void CaptureViewElement::RequestUpdate() {
+  if (parent_ != nullptr) {
+    return parent_->RequestUpdate();
```

Comment:
You're returning `void`, I assume this is not intended?

Reply:
oh, thanks! Yeah, of course

Reply:
Actually it was before, I didn't see it.